pod.go

```diff
@@ -242,11 +242,11 @@ type PodConfig struct {
 	NetworkModel  NetworkModel
 	NetworkConfig NetworkConfig
 
-	// Rootfs is the pod root file system in the host.
-	// This can be left empty if we only have a set of containers
-	// workload images and expect the agent to aggregate them into
-	// a pod from the guest.
-	RootFs string
+	// Interactive specifies if the pod runs in the foreground.
+	Interactive bool
+
+	// Console is a console path provided by the caller.
+	Console string
```

Comment:
It seems to me we're mixing 2 different changes from the same commit, which makes things a little difficult to review, and will not look nice on our git history.
Could you please separate things out a little better ?